README.md

```diff
@@ -855,6 +855,7 @@ _Related: [Groupware](#groupware)_
 
 **[`^        back to top        ^`](#)**
 
+- [GarageHQ](https://garagehq.deuxfleurs.fr/) - An open-source distributed storage service you can self-host to fullfill many needs - S3 compatible. ([Source Code](https://git.deuxfleurs.fr/Deuxfleurs/garage)) `AGPL-3.0` `Rust`
```

Comment:
```suggestion
- [GarageHQ](https://garagehq.deuxfleurs.fr/) - An open-source geo-distributed storage service you can self-host to fulfill many needs - S3 compatible. ([Source Code](https://git.deuxfleurs.fr/Deuxfleurs/garage)) `AGPL-3.0` `Rust`
```

Reply:
Just fixed a typo + added the `geo-` prefix. It is really important feature of Gearage, that they trey to address multi-dc scenario, while interconnects can be subject to different latencies (and have different throughput).